integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/createTable/TestCreateTableWithColumnMetCacheAndCacheLevelProperty.scala

```diff
@@ -127,6 +127,12 @@ class TestCreateTableWithColumnMetCacheAndCacheLevelProperty extends QueryTest w
     assert(isExpectedValueValid("default", "column_meta_cache", "column_meta_cache", "c1,c3"))
   }
 
+  test("validate for binary - COLUMN_META_CACHE_13") {
```

Comment:
As discussed for above COLUMN_META_CACHE, no need to support for binary columns.

Reply:
removed